code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/** HEADER **/
+
+header {
+  padding: 10px 20px;
+  background: #FFFFFF;
+  color: #121212;
+  font-family: Helvetica;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  header {
+    justify-content: space-evenly;
+  }
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  /*to spread the lines, since they are stacked*/
+  justify-content: space-between;
+  height: 12px;
+  width: 18px;
+  margin: 0px, 0px, 0px, -80px;
+}
+
+.hamburger div {
+  width: 18px;
+  height: 2px;
+  background-color: #121212;
+}
+
+@media (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+}
+
+header img {
+  width: 130px;
+  height: 30.42px;
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  .header-image {
+    display: none;
+  }
+}
+
+/** NAVIGATION BAR**/
+nav {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  nav {
+    display: flex;
+    align-items: center;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav-image {
+    display: none;
+  }
+  nav a:nth-last-of-type(3) {
+    display: none;
+    /*had to remove <a>; there was space left if only <img> was removed*/
+  }
+}
+
+nav a {
+  color: #121212;
+  text-decoration: none;
+  margin: 0px 12px;
+}
+
+main {
+  padding: 0px 20px;
+}
+
+@media (min-width: 1024px) {
+  main {
+    padding: 0px 30px;
+  }
+}
+
+/** BIG NEWS BOX**/
+
+.big-news {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.big-news img {
+  width: 100%;
+  height: 200px;
+}
+
+@media (min-width: 768px) {
+  .big-news img {
+    width: 45%;
+    height: 90%;
+    align-self: center;
+  }
+}
+
+.big-news-article {
+  width: 100%;
+  height: 200px;
+  padding: 5px;
+  font-family: 'Playfair Display', serif;
+}
+
+@media (min-width: 768px) {
+  .big-news-article {
+    width: 45%;
+    align-self: center;
+  }
+}
+
+.big-news-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  padding: 14px;
+}
+
+@media (min-width: 768px) {
+  .big-news-card {
+    flex-wrap: nowrap;
+    width: 100%;    
+    justify-content: space-evenly;
+  }
+}
+
+@media (min-width: 1024px) {
+  .big-news-card {
+    width: 100%;
+  }
+}
+
+.big-news-card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.big-news-article a {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 14px;
+  text-align: left;
+  color: #DB3334;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  text-transform: uppercase;
+  font-size: 18px;
+  text-align: left;
+  color: #000000;
```

Comment:
In hexadecimal approach, if all 6 parts of a value are the same, we can just type 3 and it will be ok, so `color: #000` is completely enough